core/src/utils/FilesystemUtils.cpp

```diff
@@ -29,19 +29,31 @@
  *
  */
 
-#include "FilesystemUtils.h"
-#include <QDir>
-#include <QDebug>
-#include <QCoreApplication>
+#include "FilesystemUtils.hpp"
 #include <iostream>
+#include <experimental/filesystem>
```

Comment:
We'll need to make sure this works on all targets, I remember that some experimental features did not compile well on some CI targets

Reply:
Yes, I am testing on linux and Mac and Windows on my side to make sure it works.

Reply:
tested and some small things are fixed in Cmake file